post_installation_configuration/post-install-vsphere-zones-regions-configuration.adoc

```diff
@@ -0,0 +1,56 @@
+:_content-type: ASSEMBLY
+:context: post-install-vsphere-zones-regions-configuration
+[id="post-install-vsphere-zones-regions-configuration"]
+=  Multiple regions and zones configuration for a cluster on vSphere
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+As an administrator, you can specify multiple regions and zones for your {product-title} cluster that runs on a VMware vSphere instance. This configuration reduces the risk of a hardware failure or network outage causing your cluster to fail.
+
+A failure domain configuration lists parameters that create a topology. The following list states some of these parameters:
+
+* `computeCluster`
+
+* `datacenter`
+
+* `datastore`
+
+* `networks`
+
+* `resourcePool`
+
+After you define multiple regions and zones for your {product-title} cluster, you can create or migrate nodes to another failure domain.
+
+[IMPORTANT]
+====
+If you want to migrate pre-existing {product-title} cluster compute nodes to a failure domain, you must define a new compute machine set for the compute node. This new machine set can scale up a compute node according to the topology of the failure domain, and scale down the pre-existing compute node. 
+
+The cloud provider adds `topology.kubernetes.io/zone` and `topology.kubernetes.io/region` labels to any compute node provisioned by a machine set resource. 
+
+For more information, see xref:../machine_management/creating_machinesets/creating-machineset-vsphere.adoc[Creating a compute machine set].
+====
+
+include::modules/specifying-regions-zones-infrastructure-vsphere.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../post_installation_configuration/post-install-vsphere-zones-regions-configuration.adoc#references-regions-zones-infrastructure-vsphere_post-install-vsphere-zones-regions-configuration[Parameters for the cluster-wide infrastructure CRD]
+
+include::modules/vsphere-enabling-multiple-layer2-networks.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../installing/installing_vsphere/installing-vsphere-network-customizations.adoc#installation-network-connectivity-user-infra_installing-vsphere-network-customizations[Network connectivity requirements]
+
+* xref:../machine_management/creating-infrastructure-machinesets.adoc[Creating infrastructure machine sets]
```

Comment:
All xrefs must have an anchor id at the end, as it is in the above lines. [Guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references)

Reply:
Hi @sheriff-rh . I updated these links, but is it possible to retain an anchor when pointing to the start of an assembly? For example: https://docs.openshift.com/container-platform/4.12/machine_management/creating_machinesets/creating-machineset-vsphere.html